### DISCUSSION TITLE
LOOKING FOR LOCK-IN BUDDIES

### DISCUSSION THREAD

**Discussion by Kuusho** [Cyfrin/foundry-full-course-cu#2884] [Ideas]
I'm struggling with really grasping the concepts we've worked through in this course and finishing exercises in Node Guardians.

To fix this, I'm starting a study group. It might look like:

- Daily Challenges
- Leaderboard (for fastest completion of challenges)
- Weekly Calls
- Group Projects

Please leave a comment if you're interested, I'll leave an update for next steps and requirements.

**Comment by EngrPips** [comment]
Awesome Idea. I wish you the best, Friend.

**Comment by 0xivanalexander** [comment]
Add me my man

**Reply by Reoyce** [reply]
i am interested in it

**Reply by Kuusho** [reply]
Drop your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Comment by EngrPips** [comment]
Interesting.

**Comment by heheboii11** [comment]
Count me in ser 🫡

**Reply by Reoyce** [reply]
So how do we start this?

**Reply by heheboii11** [reply]
A telegram group or may be a discord ?

**Reply by Kuusho** [reply]
Telegram group for a start

**Comment by JudeAnumudu** [comment]
hey, am interested!

**Reply by EngrPips** [reply]
Awesome.

**Reply by Kuusho** [reply]
Drop your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Comment by victor-ikeme** [comment]
who is ready, i have my solar inverter and Starlink herte in lagos Nigeria.... f laziness and poverty

Who wants to lockin from now till January? Cyfrin , solidity and more??? who? hit me up on telegram @Kingsman00z

**Reply by EngrPips** [reply]
Please how much does the starlink subscription cost monthly, I am considering it

**Reply by Kuusho** [reply]
Done

**Reply by yawarasuuna** [reply]
Keep going, brother. Nigeria will be a powerhouse in web3

**Reply by hussain-analyst** [reply]
Over here from Nigeria

**Comment by Ifra001** [comment]
I am interested as well! Working as a team really is motivating for me

**Reply by Kuusho** [reply]
Reply with your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Comment by leticarolina** [comment]
I'm interested to join!

**Reply by Kuusho** [reply]
Send over your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Reply by leticarolina** [reply]
Guys add me in! My telegram is: itisleticia

**Reply by Kuusho** [reply]
Done

**Comment by GeneralPharoah** [comment]
Nice idea bro. I'm interested. Add me up.

**Reply by Kuusho** [reply]
Drop your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Comment by johan784** [comment]
count me in!

**Reply by Kuusho** [reply]
Drop your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Comment by tcrin** [comment]
Great idea, I'm interested

**Reply by Kuusho** [reply]
Love the Law profile picture, Drop your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Comment by shoaib-eth** [comment]
I'm Interested

**Reply by Kuusho** [reply]
Drop your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Reply by shoaib-eth** [reply]
t.me/shoaibeth

**Reply by Kuusho** [reply]
Done

**Reply by Kelechikizito** [reply]
https://t.me/kelechiikizito

**Comment by mannykings2** [comment]
Fvck it, we ball. Let's do this

**Reply by Kuusho** [reply]
Drop your telegram handle please, also check this https://github.com/Cyfrin/foundry-full-course-cu/discussions/2884#discussioncomment-11359783

**Comment by EngrPips** [comment]
Great energy here.

**Reply by Kuusho** [reply]
I KNOW RIGHT! You should jump in as well

**Reply by EngrPips** [reply]
> I KNOW RIGHT! You should jump in as well

Yeah, I will as soon as I can.

**Comment by Kuusho** [comment]
> I'm struggling with really grasping the concepts we've worked through in this course and finishing exercises in Node Guardians.
> 
> To fix this, I'm starting a study group. It might look like:
> 
> * Daily Challenges
> * Leaderboard (for fastest completion of challenges)
> * Weekly Calls
> * Group Projects
> 
> Please leave a comment if you're interested, I'll leave an update for next steps and requirements.

**RE: LOCK-IN BUDDIES**

For starters, there are two baseline requirements:
1. Telegram (all comms will be handled here, via a telegram group.
2. A minimum of completing the DeFi Project of the cyfrin advanced foundry course (will check via a public repo on github).

For now I'll be adding everyone directly so leave your TG handle in the replies if you're up for it.

**Reply by tcrin** [reply]
1. Telegram handle: @nir_tc
2. I haven't completed the DeFi project of the Cyfrin Advanced Foundry course yet, but I will finish it soon.

Looking forward to joining everyone!

**Reply by sjayofweb3** [reply]
> > I'm struggling with really grasping the concepts we've worked through in this course and finishing exercises in Node Guardians.
> > To fix this, I'm starting a study group. It might look like:
> > 
> > * Daily Challenges
> > * Leaderboard (for fastest completion of challenges)
> > * Weekly Calls
> > * Group Projects
> > 
> > Please leave a comment if you're interested, I'll leave an update for next steps and requirements.
> 
> **RE: LOCK-IN BUDDIES**
> 
> For starters, there are two baseline requirements:
> 
> 1. Telegram (all comms will be handled here, via a telegram group.
> 2. A minimum of completing the DeFi Project of the cyfrin advanced foundry course (will check via a public repo on github).
> 
> For now I'll be adding everyone directly so leave your TG handle in the replies if you're up for it.

@k1ngsjay tg handle

**Reply by Ifra001** [reply]
sorry bro, don't have telegram and my country doesn't allow access to it either :(
Really wanted to join

**Reply by heheboii11** [reply]
> > I'm struggling with really grasping the concepts we've worked through in this course and finishing exercises in Node Guardians.
> > To fix this, I'm starting a study group. It might look like:
> > 
> > * Daily Challenges
> > * Leaderboard (for fastest completion of challenges)
> > * Weekly Calls
> > * Group Projects
> > 
> > Please leave a comment if you're interested, I'll leave an update for next steps and requirements.
> 
> **RE: LOCK-IN BUDDIES**
> 
> For starters, there are two baseline requirements:
> 
> 1. Telegram (all comms will be handled here, via a telegram group.
> 2. A minimum of completing the DeFi Project of the cyfrin advanced foundry course (will check via a public repo on github).
> 
> For now I'll be adding everyone directly so leave your TG handle in the replies if you're up for it.

Tg username: @Expressboii

**Reply by Kuusho** [reply]
Done

**Reply by abhinavreddyyy** [reply]
> > I'm struggling with really grasping the concepts we've worked through in this course and finishing exercises in Node Guardians.
> > To fix this, I'm starting a study group. It might look like:
> > 
> > * Daily Challenges
> > * Leaderboard (for fastest completion of challenges)
> > * Weekly Calls
> > * Group Projects
> > 
> > Please leave a comment if you're interested, I'll leave an update for next steps and requirements.
> 
> **RE: LOCK-IN BUDDIES**
> 
> For starters, there are two baseline requirements:
> 
> 1. Telegram (all comms will be handled here, via a telegram group.
> 2. A minimum of completing the DeFi Project of the cyfrin advanced foundry course (will check via a public repo on github).
> 
> For now I'll be adding everyone directly so leave your TG handle in the replies if you're up for it.

Will be completing very soon!
here's my telegram handle: @Abhinavreddyyy

**Reply by Kuusho** [reply]
Great! Just let me know once you're done but also send your tg handle

**Reply by onlyoneAlexia** [reply]
intrested

**Reply by Kuusho** [reply]
Send over your TG

**Reply by tahahasan897** [reply]
Hi, my TG: @tahahasan897
PLEASE ADD ME!!!

**Comment by wisdomtheprotector16** [comment]
Bro am wisdom Ibrahim at uniilorin let's do this

**Comment by sjayofweb3** [comment]
Bout Discord or slack?

**Comment by sjayofweb3** [comment]
Or WhatsApp?

**Reply by Ifra001** [reply]
I do have discord and whatsapp

**Comment by sjayofweb3** [comment]
That  could work then

**Comment by EngrPips** [comment]
Great energy here.

**Reply by Ifra001** [reply]
woohooo!

**Comment by SamWeb3NG** [comment]
just hit me up on telegram im interested man...my TG is @samtech2023

**Comment by JudeAnumudu** [comment]
here is my tg: https://t.me/Jweb31

**Comment by Emmyt24** [comment]
i'm interested pls add me
my TG is
@Paps002

**Comment by finestxxx** [comment]
Hi, i am interested... pls add up. my TG- @finestxxx

**Comment by Kuusho** [comment]
> I'm struggling with really grasping the concepts we've worked through in this course and finishing exercises in Node Guardians.
> 
> To fix this, I'm starting a study group. It might look like:
> 
> * Daily Challenges
> * Leaderboard (for fastest completion of challenges)
> * Weekly Calls
> * Group Projects
> 
> Please leave a comment if you're interested, I'll leave an update for next steps and requirements.

UPDATE: We've decided to open up the TG for a few more people over the next couple of days (open till 2nd January) to onboard a few more people gradually. 

The goal now is to set Q1 goals and possibly reward completion just to incentivize learning as a group. This is still an experiment 🙂

Join here: https://t.me/+ps47SvxHrjw0YjE0

**Reply by EngrPips** [reply]
Interesting. I trust you guys will crush those goals.

**Comment by hussain-analyst** [comment]
Here's my username @NigerianWeb3

**Comment by onlyoneAlexia** [comment]
Hey hey, right here

**Reply by EngrPips** [reply]
Welcome aboard.

**Comment by bonalarahul** [comment]
I am interested to join

**Reply by EngrPips** [reply]
Interesting.

**Comment by Rohaan-Taneja** [comment]
I am interested in joining in .

**Comment by tahahasan897** [comment]
Hi, my TG: @tahahasan897
PLEASE ADD ME!!!

**Comment by Kelechikizito** [comment]
> I'm struggling with really grasping the concepts we've worked through in this course and finishing exercises in Node Guardians.
> 
> To fix this, I'm starting a study group. It might look like:
> 
> * Daily Challenges
> * Leaderboard (for fastest completion of challenges)
> * Weekly Calls
> * Group Projects
> 
> Please leave a comment if you're interested, I'll leave an update for next steps and requirements.

Hello I have completed the DeFi Section, This is my TG Handle: https://t.me/kelechiikizito

**Comment by 0xebby** [comment]
up for this, here's my tg == onchain_ebby

**Comment by Abhishek-2416** [comment]
Saw this today wanted to check in how is the progress going, I am interested if this is still on

**Comment by pazzo2005** [comment]
I am interested , if this is still going on

**Comment by Joshkovu** [comment]
I am interested if it is still going on

**Comment by AryanPandeyDev** [comment]
hey i'm interested
here's my telegram:  https://t.me/cordin13

**Comment by Baxonite** [comment]
intrested.

**Comment by CreativityofWeb3** [comment]
hope it's not too late? Add me up
